folly/Subprocess.cpp

```diff
@@ -313,8 +313,11 @@ void Subprocess::spawn(
 // With -Wclobbered, gcc complains about vfork potentially cloberring the
 // childDir variable, even though we only use it on the child side of the
 // vfork.
+
+#if defined(__GNUC__) && !defined(__clang__)
```

Comment:
Move this to the next line. Only need the `__clang__` check.

Reply:
Ah, good point. Will fix now.